app/models/user.rb

```diff
@@ -35,6 +35,10 @@ def splitwise_secret=(text)
     super(self.encrypt(text))
   end
 
+  def authenticated_with_splitwise
+    return !self.splitwise_secret.nil? && !self.splitwise_token.nil?
```

Comment:
Creo que los selfs no son necesarios. :thinking: 
```suggestion
    return !splitwise_secret.nil? && !splitwise_token.nil?
```

Reply:
yo tengo entendido que si!